db/backend_test.go

```diff
@@ -0,0 +1,43 @@
+package db
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	common "github.com/tendermint/go-common"
+)
+
+func print(x []byte) {
```

Comment:
don't see where it's being used...

Reply:
Didn't this repo die long ago? To be reborn as tmlibs